src/plugins/intel_gna/CMakeLists.txt

```diff
@@ -41,6 +41,11 @@ file(GLOB_RECURSE HEADERS
         ${CMAKE_CURRENT_SOURCE_DIR}/src/*.h
         ${CMAKE_CURRENT_SOURCE_DIR}/src/*.hpp)
 
+# build avx2.cpp with AVX2 support
+ie_avx2_optimization_flags(avx2_flags)
+set_source_files_properties(${CMAKE_CURRENT_SOURCE_DIR}/avx2.cpp PROPERTIES COMPILE_OPTIONS "${avx2_flags}")
+add_compile_definitions(HAVE_AVX2=1)
```

Comment:
I suppose we should use if under:
```cmake
if(ENABLE_AVX2)
   ...
endif()
```

If we compile without AVX2, we have tor `avx2.cpp` from a list of source files